rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -546,6 +546,9 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 		}
 	}
 
+	m_emu_settings->EnhanceComboBox(ui->shaderPrecision, emu_settings_type::ShaderPrecisionQuality);
```

Comment:
Lol there's literally comment for the combo box section below 🤣

Reply:
There's also one above.

Reply:
well. you can remove the comment below then

Reply:
Removed.